phoenix-core/src/main/java/org/apache/phoenix/jdbc/LoggingPhoenixStatement.java

```diff
@@ -62,4 +75,10 @@ public ResultSet getGeneratedKeys() throws SQLException {
         return new LoggingPhoenixResultSet(super.getGeneratedKeys(), phoenixMetricsLog, this.sql);
     }
 
+    private void loggingAutoCommitHelper() throws SQLException {
+        if(conn instanceof LoggingPhoenixConnection && conn.getAutoCommit()){
```

Comment:
nit: 
just change the order of conditions
add brackets around `conn instanceof LoggingPhoenixConnection`
 and add spaces before brackets in conditions (applicable everywhere)